root/edit/details/RemoveRelationshipType.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import IntentionallyRawIcon from '../components/IntentionallyRawIcon';
+import {ENTITY_NAMES} from '../../static/scripts/common/constants';
+
+type RemoveRelationshipTypeEditT = {
+  ...EditT,
+  +display_data: {
+    +attributes: $ReadOnlyArray<{
+      ...LinkTypeAttrTypeT,
+      +typeName: string,
+    }>,
+    +description: string | null,
+    +entity0_type: CoreEntityTypeT,
+    +entity1_type: CoreEntityTypeT,
+    +link_phrase: string,
+    +long_link_phrase: string,
+    +name: string,
+    +reverse_link_phrase: string,
+  },
+};
+
+type Props = {
+  +edit: RemoveRelationshipTypeEditT,
+};
+
+const RemoveRelationshipType = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const entity0Type = ENTITY_NAMES[display.entity0_type]();
+  const entity1Type = ENTITY_NAMES[display.entity1_type]();
+
+  // Always display entity placeholders for ease of understanding
+  let longLinkPhrase = display.long_link_phrase;
+  if (longLinkPhrase && !longLinkPhrase.match('{entity0}')) {
```

Comment:
(match is meant to be passed a regular expression)
```suggestion
  if (longLinkPhrase && !longLinkPhrase.includes('{entity0}')) {
```

Reply:
Made the change, and also on `AddRelationshipType`